components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m

```diff
@@ -0,0 +1,205 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialTextControls+BaseTextAreas.h"
+#import "MaterialTextControlsPrivate+BaseStyle.h"
+
+@interface MDCBaseTextArea (Private)
+- (BOOL)shouldLayoutForRTL;
+@end
+
+@interface MDCBaseTextAreaTests : XCTestCase
+@end
+
+@implementation MDCBaseTextAreaTests
+
+- (void)testFloatingLabelColorAccessors {
+  // Given
+  CGRect textAreaFrame = CGRectMake(0, 0, 130, 40);
+  MDCBaseTextArea *textArea = [[MDCBaseTextArea alloc] initWithFrame:textAreaFrame];
+  UIColor *floatingLabelColorNormal = [UIColor blueColor];
```

Comment:
I think as a convention we typically use the dot syntax. Here and throughout.

Reply:
Done